docs/setup/custom-config.mdx

```diff
@@ -5,45 +5,19 @@ sidebarTitle: Extend the Default MindsDB Configuration
 
 To follow this guide, install MindsDB locally via [Docker](/setup/self-hosted/docker-desktop) or [PyPI](/setup/self-hosted/pip/source).
 
-## Starting MindsDB with Default Configuration
-
-Start MindsDB locally with the default configuration.
-
-1. Activate the virtual environment:
-
-```bash
-source mindsdb/bin/activate
-```
-
-2. Start MindsDB:
-
-```bash
-python -m mindsdb
-```
-
-3. Access MindsDB locally at `127.0.0.1:47334`.
-
-<Tip>
-By default, MindsDB starts the `http` and `mysql` APIs. You can define which APIs to start using the `api` flag as below.
-
-```bash
-python -m mindsdb --api http,mysql,postgres
-```
-
-If you want to start MindsDB without the graphical user interface (GUI), use the `--no_studio` flag as below.
-
-```bash
-python -m mindsdb --no_studio
-```
-</Tip>
-
 ## Starting MindsDB with Extended Configuration
 
 Start MindsDB locally with your custom configuration by providing a path to the `config.json` file that stores custom config parameters listed in this section.
 
-```bash
-python -m mindsdb --config=/path-to-the-extended-config-file/config.json
-```
+<CodeGroup>
+    ```bash Docker
+    docker run --name mindsdb_container -e MINDSDB_CONFIG_PATH=/Users/username/path/config.json -e MINDSDB_APIS=http,mysql -p 47334:47334 -p 47335:47335 mindsdb/mindsdb
+    ```
+
+    ```bash Python
+    python -m mindsdb --config=/path-to-the-extended-config-file/config.json
```

Comment:
may be add `--api=http,mysql` to make it similar to docker command above?

Reply:
thanks. added it.